src/Whoops/Handler/PrettyPageHandler.php

```diff
@@ -98,6 +111,10 @@ public function __construct()
 
         // Add the default, local resource search path:
         $this->searchPaths[] = __DIR__ . "/../Resources";
+        
+        // blacklist php provided auth based values
+        $this->blacklist('_SERVER', 'PHP_AUTH_USER');
```

Comment:
In my opinion, user portion is not that sensitive. In addition, because we do not have an API to unblacklist, it is easy to annoy existing users of Whoops. Thus I advice to remove `PHP_AUTH_USER` from the default blacklist, but keep `PHP_AUTH_PW`.